portfolio/src/main/webapp/script.js

```diff
@@ -31,10 +31,27 @@ function addRandomGreeting() {
  * Adds random fact about me to the page
  */
 function addRandomFact() {
+  // Keep track of how many times the function has been called, to determine
+  // when to reveal the quiz
+  if (typeof addRandomFact.numTimesCalled === 'undefined') {
```

Comment:
nit - does 'undefined' need to be quoted? I think the check is stronger without the quotes...

Reply:
I don't believe the quotes are necessary, I'll remove the quotes.